app/models/hackney/work_order.rb

```diff
@@ -102,4 +109,40 @@ def is_dlo?
     supplier_reference.present? or raise "is_dlo?: API responded with blank supplier reference"
     !!(/\AH\d\d\z/ =~ supplier_reference)
   end
+
+  def tasks
+    @tasks ||= Hackney::Task.for_work_order(reference)
+  end
+
+  private
+
+  def self.attributes_from_api(api_attributes)
+    # FIXME: is this still necessary?
+    stripped = api_attributes
```

Comment:
where do you use `stripped`

Reply:
oops it should be used on the following lines

Reply:
fixed!

Reply:
Instead of converting each pair to an `array` and then converting back to a `hash`, you could use [transform_values](https://ruby-doc.org/core-2.5.1/Hash.html#method-i-transform_values)

Reply:
cool! I'll do that!